ArchiSteamFarm/Bot.cs

```diff
@@ -635,6 +649,80 @@ internal sealed class Bot : IDisposable {
 			return await ArchiWebHandler.GetTradeHoldDurationForTrade(tradeID).ConfigureAwait(false);
 		}
 
+		internal async Task<OrderedDictionary> GetUsedKeys() {
```

Comment:
Used keys have no particular order, you should use generic `Dictionary<string, string>` instead of this awful `OrderedDictionary` that I had to use for redeeming process.